### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by faisalshaik78** [123/scriptaculous#4]
const apiKey = 'YOUR_API_KEY_HERE'; // Replace with your OpenWeatherMap API key
   const searchBtn = document.getElementById('search-btn');
   const cityInput = document.getElementById('city-input');
   const weatherInfo = document.getElementById('weather-info');

   searchBtn.addEventListener('click', () => {
       const city = cityInput.value;
       if (city) {
           fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}&units=metric`)
               .then(response => response.json())
               .then(data => {
                   if (data.cod === 200) {
                       weatherInfo.innerHTML = `
                           <h2>${data.name}, ${data.sys.country}</h2>
                           <p>Temperature: ${data.main.temp}°C</p>
                           <p>Weather: ${data.weather[0].description}</p>
                           <p>Humidity: ${data.main.humidity}%</p>
                       `;
                   } else {
                       weatherInfo.innerHTML = '<p>City not found. Try again.</p>';
                   }
               })
               .catch(error => console.error('Error:', error));
       }
   });

**Comment by faisalshaik78** [comment]
import tensorflow as tf
from tensorflow import keras

# Load dataset (e.g., MNIST for digit recognition)
(x_train, y_train), (x_test, y_test) = keras.datasets.mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0

# Build model
model = keras.Sequential([
    keras.layers.Flatten(input_shape=(28, 28)),
    keras.layers.Dense(128, activation='relu'),
    keras.layers.Dense(10, activation='softmax')
])

# Compile and train
model.compile(optimizer='adam', loss='sparse_categorical_crossentropy', metrics=['accuracy'])
model.fit(x_train, y_train, epochs=5)

# Evaluate
model.evaluate(x_test, y_test)
